specs/latest/1.0/index.html

```diff
@@ -4090,10 +4090,24 @@ <h2 class="no-toc">Table of contents</h2>
 <h3>Fragment shader output</h3>
 
 <p>
-    If a fragment shader writes to neither <code>gl_FragColor</code> nor <code>gl_FragData</code>,
-    the values of the fragment colors following shader execution are untouched.
+    If a fragment statically uses neither <code>gl_FragColor</code> nor <code>gl_FragData</code>, it
+    behaves as if (0, 0, 0, 0) would be assigned to <code>gl_FragColor</code> in the beginning of
+    the shader.
 </p>
 
+<p>
+    Statically used fragment output variables default to zero if they are not written during a
```

Comment:
Easier to say: "All fragment shader outputs will default to zeros."

Reply:
Yes. They will behave as if they default to zero, but that's different from us writing zeros into both of them. (which would break the mutually-exclusive static-use requirement)
The wording you have right now is technically correct, but requires more knowledge to piece together that this functionally guarantees that whichever output you choose, it will default to zero. Easier to just say they all default to zero.